src/test/java/de/uhd/ifi/se/decision/management/jira/extraction/decxtracteventlistener/TestEventCommentAdded.java

```diff
@@ -0,0 +1,69 @@
+package de.uhd.ifi.se.decision.management.jira.extraction.decxtracteventlistener;
+
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.atlassian.jira.event.issue.IssueEvent;
+import com.atlassian.jira.event.type.EventType;
+import com.atlassian.jira.issue.comments.Comment;
+
+import de.uhd.ifi.se.decision.management.jira.TestSetUpWithIssues;
+import de.uhd.ifi.se.decision.management.jira.model.DecisionKnowledgeElement;
+import de.uhd.ifi.se.decision.management.jira.model.KnowledgeType;
+import net.java.ao.test.jdbc.Data;
+import net.java.ao.test.junit.ActiveObjectsJUnitRunner;
+
+@RunWith(ActiveObjectsJUnitRunner.class)
+@Data(TestSetUpWithIssues.AoSentenceTestDatabaseUpdater.class)
+public class TestEventCommentAdded extends TestSetUpEventListener {
+
+	@Test
+	public void testNoCommentContained() {
+		Comment comment = createComment("");
+		IssueEvent issueEvent = createIssueEvent(comment, EventType.ISSUE_COMMENTED_ID);
+		listener.onIssueEvent(issueEvent);
+		assertTrue(checkComment(""));
+
+		DecisionKnowledgeElement element = getFirstElementInComment(comment);
+		assertTrue(element == null);
+	}
+
+	@Test
+	public void testRationaleTag() {
+		Comment comment = createComment("{issue}This is a very severe issue.{issue}");
+		IssueEvent issueEvent = createIssueEvent(comment, EventType.ISSUE_COMMENTED_ID);
+		listener.onIssueEvent(issueEvent);
+
+		assertTrue(checkComment("{issue}This is a very severe issue.{issue}"));
+		DecisionKnowledgeElement element = getFirstElementInComment(comment);
```

Comment:
@dekome , these tests sometimes succeed and sometimes fail. Can you please find out why they fail and fix that? Thanks.